extension/js/common/platform/store/contact-store.ts

```diff
@@ -130,7 +130,7 @@ export class ContactStore extends AbstractStore {
         throw new Error(`Could not read pubkey as valid OpenPGP key for: ${validEmail}`);
       }
       const keyDetails = await PgpKey.details(k);
-      const expiresOnMs = Number(await PgpKey.dateBeforeExpirationIfAlreadyExpired(k)) || undefined;
+      const expiresOnMs = Number(pubkey) || undefined;
```

Comment:
This looks incorrect, unless I missed something

Reply:
Yes, definitely. Thanks for noticing!